source/css/_objects.blocks.scss

```diff
@@ -94,3 +111,50 @@
     margin-left: $space;
   }
 }
+
+/**
+ * Media Block
+ */
+
+ .media-block {
+   display: flex;
+   flex-direction: row;
+
+   .block__content {
+     display: flex;
+     justify-content: space-between;
+     flex-direction: column;
+     flex-wrap: wrap;
```

Comment:
The title and dek should be wrapped and align `flex-start`. The meta content should be `flex-end`.
![img](http://d.pr/i/BQhNNE+)